### PR TITLE
Fix typo in ruff config filename

### PR DESCRIPTION
**Pull request by vtnate**

### CODE REVIEW

File: urbanopt_des/measures/set_argument_value/measure.py [hunk 1]
```diff
@@ -32,18 +32,10 @@ def arguments(self):
...
-        self.measure_args.add_argument(
-            "identifier", "Name of the type identifier being modified", units="string"
-        )
-        self.measure_args.add_argument(
-            "argument_name", "Name of the argument to set", units="string"
-        )
-        self.measure_args.add_argument(
-            "value", "Value to set the argument to", units="float"
-        )
+        self.measure_args.add_argument("type", "Name of the type being modified", units="string")
```
Comment (nllong) [inline urbanopt_des/measures/set_argument_value/measure.py:35]: this is so much better! Thanks!

File: urbanopt_des/urbanopt_analysis.py [hunk 2]
```diff
@@ -398,17 +324,17 @@ def resample_actual_data(self) -> None:
             "converted_units",
         ]
         new_data = self.actual_data_monthly.groupby(groupby_cols).sum()
-        new_data.reset_index(inplace=True)
-        new_data.set_index(["start_time"], inplace=True)
+        new_data = new_data.reset_index()
```
Comment (nllong) [inline urbanopt_des/urbanopt_analysis.py:327]: inplace is no longer needed? cool.... I learned something.

File: urbanopt_des/urbanopt_analysis.py [hunk 3]
```diff
@@ -633,26 +545,20 @@ def create_modelica_aggregations(self) -> None:
...
+            building_aggs["Total Thermal Energy Heating"]["agg_columns"] = ["Total Thermal Heating Energy"]
         finally:
             pass
 
         # Do this for each of the analyses' 15 and 60 min dataframes
-        for analysis_name in self.modelica.keys():
+        for analysis_name in self.modelica:
             for resolution in ["min_15_with_buildings", "min_60_with_buildings"]:
-                df = getattr(self.modelica[analysis_name], resolution)
+                temp_df = getattr(self.modelica[analysis_name], resolution)
```
Comment (nllong) [inline urbanopt_des/urbanopt_analysis.py:561]: haha, yes yes, I know.

**Review by nllong — APPROVED** [review]
did I really have this as `rull.toml`?   😶

**Comment by vtnate** [comment]
@nllong It's all ruff! Thank goodness for linting. I just do what it says.

**Pull request merged**